### PR TITLE
hopefully fixes missing include file

### PR DESCRIPTION
**Pull request by ToMadoRe**

### CODE REVIEW

**Comment by ToMadoRe** [comment]
just referencing https://github.com/strands-project/v4r_ros_wrappers/issues/18

File: segment_and_classify/CMakeLists.txt [hunk 1]
```diff
@@ -9,8 +9,9 @@ catkin_package(
    CATKIN_DEPENDS segmentation_srv_definitions classifier_srv_definitions object_classifier segmentation
 )
 
+set(incs include/pcl_segmentation_ros.h include/pcl_conversions.h)
 include_directories(include ${catkin_INCLUDE_DIRS} ${V4R_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS})
-add_executable(segement_and_classify_test src/test.cpp)
+add_executable(segement_and_classify_test src/test.cppi ${incs})
```
Comment (marc-hanheide) [inline segment_and_classify/CMakeLists.txt]: I think you added a spurious "i" here...

**Comment by marc-hanheide** [comment]
just wondering: Why are you including `pcl_conversions.h` here and not use the existing ros package `pcl_conversions` instead?

**Comment by ToMadoRe** [comment]
good question... didn't know there is one. I will leave it for now but try to use the packaged version in the future

**Comment by marc-hanheide** [comment]
I think you should be using the `pcl_conversion` package instead. Then also this file will be available:

```
CMake Error at monitored_vcs/segment_and_classify/CMakeLists.txt:14 (add_executable):
  Cannot find source file:

    include/pcl_segmentation_ros.h

```

which makes the current build fail

**Comment by ToMadoRe** [comment]
Okay, I changed the code such that it uses the packaged version of pcl_conversions. Unfortunately, I can't test it right now as I am away and don't have a Kinect with me.

**Comment by marc-hanheide** [comment]
OK, but now it compiles... That's progress already, so I'll re-release this (fingers crossed again)

**Pull request merged**